src/common/SearchBar.js

```diff
@@ -49,6 +49,7 @@ const SearchBar = () => {
       <StyledInputBase
         placeholder="Search…"
         inputProps={{ 'aria-label': 'search' }}
+        onChange={(e) => onQueryChange(e.target.value)}
```

Comment:
We probably don't want to run the query on every change, but rather on a button click, so the search term state should be here.